FreeRTOS/Demo/RISC-V-Qemu-virt_GCC/main.c

```diff
@@ -30,6 +30,10 @@
 
 /* Run a simple demo just prints 'Blink' */
 #define DEMO_BLINKY	1
+#define mainVECTOR_MODE_DIRECT	1
+
+extern void freertos_risc_v_trap_handler( void );
+extern void freertos_vector_table( void );
```

Comment:
Where is your `freertos_vector_table` coming from? It looks typical to define this vector_table in an assembly file (see [this file for example](https://github.com/FreeRTOS/FreeRTOS/blob/4255ac5c28af626ea7953006a4ff112f758ea332/FreeRTOS/Demo/RISC-V_RV32_QEMU_VIRT_GCC/vector.S)).

Reply:
You are right, I forgot to push the assembly file, i was focused to get the demo working in direct mode and forgot to push the file with the vector definition. 
Sorry for my mistake, how can I fix that?

Reply:
No sweat! Should be as simple as making another commit with the assembly file and pushing it to this PR.

Reply:
now is ok, sorry for the mess, never done a PR before

Reply:
The fix is been actually taken from the example you posted btw, so it should work in the same way.

Reply:
>sorry for the mess, never done a PR before

No need to be sorry. Having multiple commits for a PR is basically standard practice. This isn't a mess.

Reply:
Awesome! Thanks for the quick update. I'm trying to build this demo. Crosstools-ng doesn't like my Mac so I'm trying on a linux machine.

Reply:
Hi, did you manage to test the fix?